Issue 43648: Study dataset reorder page jumps to top when there are enough datasets to make the page include a scrollbar

cnathe opened the pull request:
#### Rationale
https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=43648

Client reported that with a 21.3.4 change it got much more difficult to reorder study datasets when there are a large number of datasets in the study (i.e. in their example, see support ticket details, they have >200 datasets). In addition to fixing the "page jump" issue, this PR adds in new "move to top" and "move to bottom" buttons to help with the client scenario.

#### Changes
* update datasetDisplayOrder.jsp to use insertBefore() and appendChilde() to move select option items 
* remove "submit" from move buttons to prevent page jump on click
* add "move to top" and "move to bottom" buttons
* similar button styling updates for Manage Visit Order and Project Display Order pages

cnathe committed:
Issue 43648: Study dataset reorder page jumps to top when there are enough datasets to make the page include a scrollbar
- also add "move to top" and "move to bottom" buttons

labkey-jeckels commented on study/src/org/labkey/study/view/datasetDisplayOrder.jsp:
```diff
@@ -115,9 +100,11 @@ function orderModule(down)
+            <td align="center" valign="center" style="padding-left: 10px;">
+                <%= button("Move Up").onClick("return moveDatasetItem('up')") %><br>
+                <%= button("Move Down").onClick("return moveDatasetItem('down')") %><br><br>
+                <%= button("Move to Top").onClick("return moveDatasetItem('top')") %><br>
```
Should we make this the first button?

cnathe replied:
I went back and forth on that. Here is a screenshot of the current implementation. I decided to group the buttons for move up/move down and then move to top / move to bottom. Happy to change it as you are suggesting if you think that is more intuitive.

<img width="566" alt="Screen Shot 2021-07-28 at 10 11 10 AM" src="https://user-images.githubusercontent.com/6411206/127357553-dcdf69c4-b675-465c-a1b7-823cc97b92d0.png">

cnathe replied:
Updated display based on chat conversation with Josh:

<img width="603" alt="Screen Shot 2021-07-28 at 1 17 21 PM" src="https://user-images.githubusercontent.com/6411206/127375057-d4f36791-3f36-42ca-b3ac-5f63978ca083.png">

labkey-jeckels replied:
Breathtaking!

cnathe committed:
Set move button widths so they aren't jagged
- and make similar updates on manage visit order and manage project display order pages
